docs/api/README.md

```diff
@@ -249,11 +249,13 @@ Usage:
 
 ## keyframes
 
-**Radium.keyframes(keyframes, componentName)**
+**Radium.keyframes(keyframes, componentName, prefix)**
```

Comment:
What are your thoughts on using the standard `(required, [optional])` pattern to denote whether a parameter is optional or not?